Sources/PointFree/Footer.swift

```diff
@@ -8,15 +8,23 @@ let footerView =
   curry(footer)([`class`([footerClass])]) >>> pure
     <¢> footerInfoColumnsView
 
+private func column(sizes: [Breakpoint: Int]) -> ([Node]) -> [Node] {
+  return gridColumn(sizes: sizes) >>> pure
```

Comment:
Been finding that `pure` inference is having a tough time. Should talk about making them `static`s in Prelude.

Reply:
yeah makes sense to me. could start with code gen'ing them both. think we should figure out a code gen sitch for us anyway.